schema/bom-1.6.proto

```diff
@@ -148,6 +148,10 @@ message Component {
   repeated OrganizationalContact authors = 29;
   // Textual strings that aid in discovery, search, and retrieval of the associated object. Tags often serve as a way to group or categorize similar or related objects by various attributes. Examples include "json-parser", "object-persistence", "text-to-image", "translation", and "object-detection".
   repeated string tags = 30;
+  // Specifies the OmniBOR Artifact ID. The OmniBOR, if specified, MUST be valid and conform to the specification defined at: https://www.iana.org/assignments/uri-schemes/prov/gitoid
+  optional string omnibor = 31;
+  // Specifies the Software Heritage persistent identifier (SWHID). The SWHID, if specified, MUST be valid and conform to the specification defined at: https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html
+  optional string swhid = 32;
```

Comment:
Same purl could have several swhid since it is based on the repository that was used to archive. For example apache projects could be mirrored in several git servers. Each of these would have same purls but different swhid.